tests/unit_tests/test_lobbyconnection.py

```diff
@@ -334,6 +341,7 @@ async def test_command_game_join_calls_join_game(
         "mod": "faf",
         "name": "Test Game Name",
         "init_mode": InitMode.NORMAL_LOBBY.value,
+        "game_type": GameType.CUSTOM.value,
```

Comment:
I think it’s better to avoid using the enum in the test assertions and put the value directly. It’s a bit more work if it changes, but it makes it more clear what we expect the message to actually look like.